precice/Dockerfile.Ubuntu1604.home

```diff
@@ -1,5 +1,11 @@
 # Dockerfile for building preCICE on ubuntu 16.04
 
+# #################
+# WARNING: As of 11.2020, images using Ubuntu 16.04 are no longer maintained.
+#          Attempting to build this dockerfile or using it as base image for
+#          other builds is likely to fail.
+# #################
```

Comment:
See above. Not really sure whether this should go into this PR.

Reply:
Fair point, I moved the comment cleanup related changes to a new PR (see https://github.com/precice/systemtests/pull/269), this PR now only handles the apt pip change. 

I force pushed my local branch here to keep the commit mess to a minimum, sorry about that :sweat_smile: At least the `git add -p` function proved to be very useful once more.